scripts/release/verify-artifacts.py

```
@@ -0,0 +1,519 @@
+        boundary = subprocess.run(
+            [
+                sys.executable,
+                str(Path(__file__).resolve().parents[1] / "check-website-boundary.py"),
+                "--build-directory",
+                str(build),
+            ],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+            check=False,
+        )
```

Ruff (0.16.1)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)